scripts/zpool

```diff
@@ -47,28 +47,49 @@ This is a resource agent that manages ZFS zpools.
 EOF
 }
 
+zpool_status_log() {
+	# Log zpool status to splunk for post-mortem.  Run in it the background
+	# in case zpool status hangs.  Wait two seconds for it to log.
+	nohup sh -c 'zpool status | logger -t resource_agent_fail -p local7.debug' &
```

Comment:
Is the main purpose of this to compare the output of zpool status, with the result from the /proc/.../state file?  If so, perhaps you should send that content to logger as well, since whatever is logged via OCF may go into a different log file or by filtered out by syslog rules.

Reply:
It's there so that we get a snapshot of what `zpool status` looked like right before the node is fenced for having a bad pool.  That way we have an idea of why the pool went bad.

Reply:
OK, that makes sense.